docs/security/configuration/tls.md

```diff
@@ -41,8 +41,9 @@ Name | Description
 
 ## Keystore and truststore files
 
-As an alternative to certificates and private keys in PEM format, you can instead use keystore and truststore files in JKS or PKCS12/PFX format. The following settings configure the location and password of your keystore and truststore files. If you want, you can use different keystore and truststore files for the REST and the transport layer.
+As an alternative to certificates and private keys in PEM format, you can instead use keystore and truststore files in JKS or PKCS12/PFX format. For the security plugin to operate, node-to-node encryption is mandatory.
```

Comment:
Can you provide _a bit_ more context here? How do these keystore and truststore settings relate to node-to-node encryption?